src/software/constants.h

```diff
@@ -5,3 +5,19 @@
 // How many milliseconds a robot must not be seen in vision before it is
 // considered as "gone" and no longer reported.
 static constexpr unsigned int ROBOT_DEBOUNCE_DURATION_MILLISECONDS = 200;
+
+
+static constexpr unsigned int MAX_SIMULATOR_MULTICAST_CHANNELS = 16;
+
+// Networking
+// the IPv6 multicast address, only ff02 is important, the rest is random
+// see https://en.wikipedia.org/wiki/Solicited-node_multicast_address for why ff02 matters
+static const std::string SIMULATOR_MULTICAST_CHANNELS[MAX_SIMULATOR_MULTICAST_CHANNELS] =
```

Comment:
smart! can't use `std::string` in fw lol

Reply:
;)